tools/RAiDER/delay.py

```diff
@@ -555,7 +553,14 @@ def build_cube_ray(xpts, ypts, zpts, ref_time, orbit_file, look_dir, model_crs,
 
 
 ###############################################################################
-def main(dt, wetFilename, hydroFilename, args):
+# def main(dt, wetFilename, hydroFilename, args, library=True):
+def main(dt, wetFilename, hydroFilename,
+            los=None, aoi=None, ray_trace=False, zref=_ZREF, los_cube=None,
+            height_levels=None, dem_file=None, use_dem_latlon=False,
+            bounding_box=None, lat_file=None, lon_file=None, hgt_file_rdr=None,
+            station_file=None, geocoded_file=None,
+            wmLoc='.', weather_model=None, cube_spacing=2000, download_only=False,
+            outformat='Gtiff', crs='EPSG:4326', look_dir='right', verbose=False):
```

Comment:
I think this is the primary (main?) deliverable.

- `main` is a pretty generic name for function - what about something more descriptive?
- for something that is exposed to the real world, specifying types would be useful
- also the return type using `->` would be helpful
- The docstring format is not something I am familiar with - but I recommend using a formatter in your IDE